cvat-ui/src/components/file-manager/remote-browser.tsx

```diff
@@ -171,6 +171,18 @@ function RemoteBrowser(props: Props): JSX.Element {
             }
         };
         collectNodes(content);
+
+        if (manifestPath) {
+            nodes.push({
+                name: manifestPath,
+                key: manifestPath,
+                children: [],
+                type: 'REG',
+                mimeType: 'unknown',
```

Comment:
Why mimeType is unknown? ``application/jsonl``?

Reply:
I was sure that `mimeType` in such a structure generally is defined from the [server content](https://github.com/opencv/cvat/blob/8962c12050ca669abf5ef497de663ead8148ae61/cvat/apps/engine/media_extractors.py#L44) and it can be only `image`/`video`/`archive`/`pdf`/`DIR` /`unknown`.